src/universal/components/AuthPage/AuthPage.js

```diff
@@ -33,6 +31,8 @@ const CenteredBlock = styled('div')({
   flexDirection: 'column',
   alignItems: 'center',
   height: '100%',
+  maxWidth: '100%',
```

Comment:
@dan-f this should take care of our overflow woes down at mobile device widths for now.